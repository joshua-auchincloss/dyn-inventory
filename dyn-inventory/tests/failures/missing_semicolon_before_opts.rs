// missing semicolon before ExtraParams
//
//
//
//
//
//
//
//
//
use dyn_inventory::dyn_inventory;

dyn_inventory!(
    MissingSemiRegistry<T: PluginApi> {
        value: i32,
        t: T,
    }
    init_name = register_missing,
);

fn main() {}
