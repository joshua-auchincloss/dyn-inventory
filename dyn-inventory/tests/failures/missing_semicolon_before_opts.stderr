error: expected `;`
  --> tests/failures/missing_semicolon_before_opts.rs:18:5
   |
18 |     init_name = register_missing,
   |     ^^^^^^^^^
